packages/cli/commands/project/dev.js

```diff
@@ -70,6 +71,13 @@ exports.handler = async options => {
 
   uiBetaTag(i18n(`${i18nKey}.logs.betaMessage`));
 
+  logger.log(
```

Comment:
Not sure if we want this here or not.  This is the initial log when `hs project dev` is ran, and this gets cleared when the account has been selected and the dev servers have started up.